homeassistant/components/homekit/type_cameras.py

```diff
@@ -434,6 +438,12 @@ def _async_stop_ffmpeg_watch(self, session_id):
         self.sessions[session_id].pop(FFMPEG_WATCHER)()
         self.sessions[session_id].pop(FFMPEG_LOGGER).cancel()
 
+    async def stop(self):
```

Comment:
Note that this is overriding a `HAP-python` method so it shouldn't be prefixed with `async_`